File: Clase6/Example2.cs
using System.IO;
public class Example{
	public static void Main(){
		FileStream fs = new FileStream("file2.txt",
		FileMode.Append,FileAccess.Write);
		StreamWriter sw = new StreamWriter(fs);
		
		while(true){
			string linea = System.Console.ReadLine();
			if(linea == null) break;
			sw.WriteLine(linea);
		}
		sw.Close();
		fs.Close();
	}
}

File: Clase6/Example2.java
import java.io.FileWriter;
import java.io.BufferedReader;
import java.io.InputStreamReader;
public class Example2{
	public static void main(String[] args) throws Exception{
		FileWriter f = new FileWriter("fileJava2.txt");
		
		BufferedReader bf = new BufferedReader
		(new InputStreamReader(System.in));
		
		
		while(true){
			String linea = bf.readLine();
			if (linea == null) break;
			f.write(linea+"\n");
			
		}
		f.close();
	}
}